reserved_test.go

```diff
@@ -10,7 +10,6 @@ func TestFetchReservedCVE(t *testing.T) {
 	tests := []struct {
 		cveID string
 	}{
-		{"CVE-2020-27949"},
```

Comment:
this CVE does not seem to be reserved (ref: https://nvd.nist.gov/vuln/detail/CVE-2020-27949), so the test was failing.